Trying to stop broadcasting when adapter is off crashes the app.

1. Open app and choose peripheral
2. Minimize app
3. Turn off bluetooth.
4. Open app again.
5. Press "Deny"

Expected: Fragment should close.

Result: App crashes.

For info, I can reproduce the error with N9 with M. Here are the logs for reference:

```
07-20 11:59:48.760  2007  2398 E AmanattoDataUpdaterHelper: package name: io.github.webbluetoothcg.bletestperipheral
07-20 11:59:48.932 10609 10609 D BluetoothAdapter: STATE_OFF
07-20 11:59:48.933 10609 10609 D AndroidRuntime: Shutting down VM
07-20 11:59:48.935 10609 10609 E AndroidRuntime: FATAL EXCEPTION: main
07-20 11:59:48.935 10609 10609 E AndroidRuntime: Process: io.github.webbluetoothcg.bletestperipheral, PID: 10609
07-20 11:59:48.935 10609 10609 E AndroidRuntime: java.lang.RuntimeException: Unable to stop activity {io.github.webbluetoothcg.bletestperipheral/io.github.webbluetoothcg.bletestperipheral.Peripheral}: java.lang.IllegalStateException: BT Adapter is not turned ON
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3788)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3842)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.app.ActivityThread.-wrap5(ActivityThread.java)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1399)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:102)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:148)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:5415)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:725)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:615)
07-20 11:59:48.935 10609 10609 E AndroidRuntime: Caused by: java.lang.IllegalStateException: BT Adapter is not turned ON
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.bluetooth.le.BluetoothLeUtils.checkAdapterStateOn(BluetoothLeUtils.java:136)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.bluetooth.le.BluetoothLeAdvertiser.stopAdvertising(BluetoothLeAdvertiser.java:155)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at io.github.webbluetoothcg.bletestperipheral.Peripheral.onStop(Peripheral.java:268)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.app.Instrumentation.callActivityOnStop(Instrumentation.java:1278)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.app.Activity.performStop(Activity.java:6305)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3783)
07-20 11:59:48.935 10609 10609 E AndroidRuntime:        ... 9 more
07-20 11:59:48.938  1004  1605 W ActivityManager:   Force finishing activity io.github.webbluetoothcg.bletestperipheral/.Peripherals

--- a/app/src/main/java/io/github/webbluetoothcg/bletestperipheral/Peripheral.java
+++ b/app/src/main/java/io/github/webbluetoothcg/bletestperipheral/Peripheral.java
@@ -262,7 +262,7 @@ public class Peripheral extends Activity implements ServiceFragmentDelegate {
     if (mGattServer != null) {
       mGattServer.close();
     }
-    if (mAdvertiser != null) {
+    if (mBluetoothAdapter.isEnabled() && mAdvertiser != null) {
       // If stopAdvertising() gets called before close() a null
       // pointer exception is raised.
       mAdvertiser.stopAdvertising(mAdvCallback);

